yw_website/yw_website/apps/website/templates/extendable/_base_detailed_workflow.html

```diff
@@ -15,45 +15,53 @@
     <div class = title>
         {{ workflow.title}}
     </div>
-    <div class = version> 
+    <div class = "version"> 
```

Comment:
for consistencies sake, lets try to write classes in html like `class="version"` instead of `class = "version"`